lib/source-destination/file.ts

```diff
@@ -135,6 +133,7 @@ export class File extends SourceDestination {
 	}
 
 	public async createWriteStream(): Promise<NodeJS.WritableStream> {
+		// @ts-ignore: @types/node is wrong about fs.WriteStream constructor: it takes 2 arguments, the first one is the file path
```

Comment:
Why ?
`createWriteStream()` returns a `ProgressWriteStream` which is an `fs.WriteStream` with some stuff added.

Reply:
Well the ProgressWriteStream surely has the right types doesn't it @zvin? In which case the `@ts-ignore` here is unnecessary and could potentially hide issues

Reply:
It does not as it is not redefining the constructor.

Reply:
Ok, I did just see in ProgressWriteStream:
```
export const ProgressWriteStream = makeClassEmitProgressEvents(
	// type definitions for node 6 export fs.WriteStream as an interface, but it's a class.
	// @ts-ignore
	fs.WriteStream,
	'bytesWritten',
	'bytesWritten',
	PROGRESS_EMISSION_INTERVAL,
);
```
it mentions an issue with node 6 typings - that can probably be updated now since you updated the node typings, it might also be worth checking if the node 8 typings fix this issue

Reply:
They're still wrong in node 13 types: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/fs.d.ts#L164

The comment was updated in another commit, it no longer mentions node 6.

Reply:
The comment about interface vs class has been removed as this one is fixed.

Reply:
Actual implementation here https://github.com/nodejs/node/blob/master/lib/internal/fs/streams.js#L286